src/utilities/mergeParams.ts

```diff
@@ -0,0 +1,19 @@
+import { Param } from '@/types'
+
+export function mergeParams(...params: Record<string, Param[]>[]): Record<string, Param[]> {
+  return params.reduce<Record<string, Param[]>>((combined, collection) => {
+    Object.entries(collection).forEach(([key, params]) => {
+      if (key in combined) {
+        params.forEach(param => {
+          if (!combined[key].includes(param)) {
```

Comment:
Referring to the test I commented on I think this condition needs to be removed.